### DISCUSSION TITLE
Mark documents as external in destination database when using ETL

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#18433] [Q&A] [answered]
We can't use document id postfix  because we have references in the documents being transfered using ETL.
I was thinking about something like this.
```js
loadToCollection({
    '@metadata': {
        '@external': true
    }
})
```
This works, but I want to add the metadata only and keep all other fields. Is this possible?

**Comment by arekpalinski** [comment, marked as answer]
You can use the following ETL script:

```
this['@metadata']["external"] = true;

loadToCollection(this);
```

Please not that I didn't use `@external: true` but `external: true`. The reason is that:

> Keys in the metadata that starting with @ are reserved for RavenDB use

https://ravendb.net/docs/article-page/6.0/csharp/client-api/session/how-to/get-and-modify-entity-metadata#modify-the-metadata
